### ISSUE TITLE
Sudden unexpected death in epilepsy

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6382]
**Preferred term label:**
Sudden unexpected death in epilepsy

**Synonyms**
SUDEP

**Definition (free text, please give PubMed ID)**
SUDEP is a sudden, unexpected, witnessed or unwitnessed, non-traumatic and non-drowning death, occurring in benign circumstances, in an individual with epilepsy, with or without evidence for a seizure and excluding documented status epilepticus, in which postmortem examination has not revealed a cause of death.

PMID 22191982

**Parent term (use hpo.jax.org/app)**
Sudden death HP:0001699

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dravet syndrome (Phenotype MIM 607208, ORPHA:33069)

**Your nano-attribution (ORCID)**
Gash Mbizvo 0000-0002-9588-2944
David Lewis-Smith 0000-0002-1735-8178

**Comment by nicolevasilevsky** [comment]
Hi @DLewisSmith 
Someone had previously requested this term and Peter did not think it was appropriate for HPO, see:
https://github.com/obophenotype/human-phenotype-ontology/issues/6210

**Comment by DLewisSmith** [comment]
Thanks @nicolevasilevsky. Hi @kearneyja and @pnrobinson. This is a difficult one for us in epileptology because despite the nomenclature and poorly understood mechanism, it is an important phenotype and formally recognised cause of death, issued by coroners/physicians/pathologists on a death certificate rather than a simply a description of the circumstances of death on a background of epilepsy and in the absence of an alternative cause (although this is what amounts to the formal cause of death).  It is a well established epidemiological entity affecting 1:1000 people with epilepsy per year, closer to 1:100 in monogenic developmental and epileptic encephalopathies.

Sadly it has been shown to be a phenotype associated with particular monogenic forms, including Dravet syndrome (cause of death in 50% of premature deaths in this most common of the monogenic developmental and epileptic encephalopathies, PMID: 27732919). It is distinct from other sudden, unexpected forms of death attributable to epilepsy (drowning, falls, motor vehicle collisions, status epilepticus), which also contribute to the excess mortality seen in people with epilepsy.

Hence, those of us looking at genetic forms of epilepsy have reason to believe in this as a phenotypic entity that is likely to be enriched in some genetic forms, and for wanting to distinguish it from other causes of sudden death that we observe in people with epilepsy.

The mechanism of death is poorly understood. I presume that this is the primary problem for inclusion in the HPO. Clearly both the mechanism (some evidence points to autonomic dysfunction of the cardiopulmonary system) and molecular associations are a major research priority. What thresholds beyond strong epidemiological association would need to be met for inclusion? I can accept that this might not be possible.

Unfortunately, most registries/studies simply record whether the participant had SUDEP rather than specific features at post-mortem (such as pulmonary oedema) that might be more appropriate to HPO coding and enlightening for discovering the mechanisms.

https://sudep.org/what-sudep

**Comment by pnrobinson** [comment]
@DLewisSmith  with this explanation, I think it is fine for an HPO term. 
Should it be a child of https://hpo.jax.org/app/browse/term/HP:0001699?

**Comment by DLewisSmith** [comment]
Yes. I think so. Thank you @pnrobinson.

**Comment by kearneyja** [comment]
Thank you, that was an elegant explanation. I am coming from committees working to advance research into SUDEP mechanisms and one of the impediments to research that we identified is a lack of HPO/MPO terms. This will be a great help in moving the field forward.

________________________________

**Comment by pnrobinson** [comment]
@DLewisSmith  added
